firefox-ios/Client/Coordinators/Router/RouteBuilder.swift

```diff
@@ -132,6 +132,7 @@ final class RouteBuilder {
                 // Pass optional share message and subtitle here
                 var shareMessage: ShareMessage?
                 if let titleText = urlScanner.value(query: "title") {
+                    // TODO: FXIOS-10629 Get the optional subtitle parameter 
```

Comment:
This is for the Info Card Referral experiment (in progress). Currently no other paths in the app use the optional ShareMessage.